addons/easy_charts/control_charts/chart.gd

```diff
@@ -71,6 +71,20 @@ func load_functions(functions: Array[Function]) -> void:
 			_:
 				function_legend.add_function(function)
 
+	_draw()
```

Comment:
@fenix-hub Not sure about this one. Why did I add this? I added buttons to add/remove bar functions to/from the bar chart example. However, when I first did this, the example crashed because the newly added `BarPlotter` have not initialized before being drawn the first time. Adding the call to `_draw()` here fixed it. I suspect the draw order being different between a newly created chart and an existing chart. This causes the functions to be drawn before the chart did initialize the plotters properly within `_draw()`.

What I'm concerned about: I'm not sure if `_draw()` does something that could break in the first call to `plot()`. Second, I'm not sure if we now initialize function plotters twice before the initial render.

Maybe you have an answer - if not, I will debug this later :)